### PR TITLE
docs: add Japanese README file

### PR DESCRIPTION
**Pull request by eltociear**
I created Japanese translated README.

### CODE REVIEW

File: readme_jp.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[English](readme.md) | [中文](readme_cn.md) | **日本語**
+
+<h1 align="center">MCDReforged PluginCatalogue</h1>
+
+<p align="center">
+  <a href="https://github.com/Fallen-Breath/MCDReforged">MCDReforged</a> プラグインを一覧表示するためのリポジトリ。
+  <br>
+  <br>
+  <a href="https://mcdreforged.com/ja/plugins"><strong>メインプラグインカタログ</strong></a>
```
Comment (Fallen-Breath) [inline readme_jp.md:9]: `mcdreforged.com` does not support `ja` language yet, you can just use `en`

File: readme_jp.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  ·
+  <a href="https://github.com/MCDReforged/PluginCatalogue/blob/legacy/readme_jp.md">旧プラグインカタログ</a>
```
Comment (Fallen-Breath) [inline readme_jp.md:11]: readme file in the `legacy` branch does not have `jp` language as well

File: readme_jp.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+> [!IMPORTANT]
+> **プラグインを使用する前に、リポジトリ内の README ファイルを必ずお読みください。**
+
+<br>
+
+**メインプラグインカタログ** には MCDReforged >= 2.x をサポートするプラグインが含まれています
+
+**旧プラグインカタログ** には以前のバージョンの MCDR 用のプラグインが含まれています。これらのプラグインは現在のバージョンの MCDR と互換性がない場合がありますので、注意して使用してください
+
+プラグインリポジトリにプラグインを追加したい場合は、PR を提出してください。詳細については [ドキュメント](https://mcdreforged.readthedocs.io/ja/latest/plugin_dev/plugin_catalogue.html) をご覧ください
```
Comment (Fallen-Breath) [inline readme_jp.md:24]: same here, no `ja` language for the docs website

**Review by alex3236 — COMMENTED** [review]
One question about this: Is it really necessary to just add a Japanese README to the PluginCatalogue when MCDReforged has no plans to provide Japanese text for other parts of the project?

**Comment by alex3236** [comment]
closing now since this has been inactive for a long while. feel free to reopen as needed

**Comment by AnzhiZhang** [comment]
> One question about this: Is it really necessary to just add a Japanese README to the PluginCatalogue when MCDReforged has no plans to provide Japanese text for other parts of the project?

This question should not be for the author, all requested changes have been applied.

**Pull request closed without merging**